lib/awful/rules.lua

```diff
@@ -1,5 +1,8 @@
 ---------------------------------------------------------------------------
---- Apply rules to clients at startup.
+--- Rules for clients.
+--
+-- This module applies @{rules} to clients during startup (via @{client.manage},
```

Comment:
`@{}` ends up as an error from LDoc if the reference cannot be found while `` just creates the same "look", but without the link in that case.